Code/Tools/AWSNativeSDKInit/source/Platform/Android/CopyCaFile_Android.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ * 
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/PlatformDef.h>
+// The AWS Native SDK AWSAllocator triggers a warning due to accessing members of std::allocator directly.
+// AWSAllocator.h(70): warning C4996: 'std::allocator<T>::pointer': warning STL4010: Various members of std::allocator are deprecated in
+// C++17. Use std::allocator_traits instead of accessing these members directly. You can define
+// _SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING or _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS to acknowledge that you have received
+// this warning.
+AZ_PUSH_DISABLE_WARNING(4251 4996, "-Wunknown-warning-option")
+#include <aws/core/utils/memory/stl/AWSString.h>
+AZ_POP_DISABLE_WARNING
+#include <AzCore/Android/Utils.h>
+#include <AzCore/IO/FileIO.h>
+#include <AzCore/IO/SystemFile.h>
+#include <AzCore/std/containers/vector.h>
+
+namespace AWSNativeSDKInit
+{
+    namespace Platform
+    {
+        void CopyCaFile()
+        {
+            AZStd::vector<char> contents;
+            AZStd::string certificatePath = "@assets@/certificates/aws/cacert.pem";

```

Comment:
As you put cacert.pem inside AWSCore gem, then why not move these AWSNativeSDKInit change into AWSCore?

Reply:
I think copy the cert bundle to the android device is part of the initialization for the SDK since you need to have the file in place to make any requests. Code/Tools/AWSNativeSDKInit is the place where all the SDK initialization code live.